src/test/resources/Homework21.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="Homework21">
+ <parameter name="BaseURL" value="https://qa.koel.app"/>
+    <test name="Homework21" preserve-order="false">
+        <classes>
+            <class name="Homework21">
+                <include name = 'playSong'/>
+            </class>
+        </classes>
+    </test>
+</suite>
```

Comment:
You got this problem, please correct it.

![image](https://github.com/Test-Pro-Bootcamp/A57-QA-Automation-Framework/assets/104761482/aaa455e6-98df-4afc-b9dd-d62e51933c90)